include/access_management.js

```diff
@@ -131,14 +131,14 @@ module.exports = function(pb) {
     };
 
     /**
-     * 
+     *
      * @static
      * @method getRoleToDisplayNameMap
      * @param {Localization} ls
      * @return {Object}
      */
     SecurityService.getRoleToDisplayNameMap = function(ls) {
-        if (util.isFunction(ls)) {
+        if (util.isFunction(ls.get)) {
```

Comment:
Why did this get changed?  There were some instances where the localization service wasn't getting passed.  The if check is there to provide support for backward compatiablity.  This change will cause a null pointer error in those cases.  Please change it back to ```if (util.isFunction(ls)) {```